default.nix

```diff
@@ -6,72 +6,126 @@ with pkgs;
 
 let
   root = ./.;
-  mainnet = {
+  mainnet = rec {
     rev = "94f779a7";
     sha256 = "16lxilng5q8fr2ll6h4hf7wlvac6nmw4cx10cbgzj5ks090bl97r";
     patchFile = ./nix/fix-mainnet.patch;
+    protoName = "005_PsBabyM1";
```

Comment:
are you sure?

Reply:
`proto` this should mean `protocol`
e.g. https://gitlab.com/tezos/tezos/tree/babylonnet/src/proto_005_PsBabyM1